pkg/pgmodel/series_set_test.go

```diff
@@ -450,93 +392,43 @@ func genRows(count int) [][][]byte {
 	return result
 }
 
-func genPgxRows(m [][][][]byte) []pgx.Rows {
+func genPgxRows(m [][]seriesSetRow, err error) []pgx.Rows {
 	result := make([]pgx.Rows, len(m))
 
 	for i := range result {
 		result[i] = &mockPgxRows{
 			results: m[i],
+			err:     err,
 		}
 	}
 
 	return result
 }
 
-func genSeries(labelNames []string, labelValues []string, ts []pgtype.Timestamptz, vs []pgtype.Float8) [][]byte {
-	result := make([][]byte, 4)
-	keys := make([]byte, 0, 100)
-	values := make([]byte, 0, 100)
-	nulls := make([]uint32, 4)
-	if len(labelNames) == 0 {
-		nulls[0] = 1
-	}
-	if len(labelValues) == 0 {
-		nulls[1] = 1
-	}
-	if len(ts) == 0 {
-		nulls[2] = 1
-	}
-	if len(vs) == 0 {
-		nulls[3] = 1
-	}
-	var err error
-	kIdx, vIdx := 0, 0
-	for _, v := range labelNames {
-		keyLen := len(v)
-		keys = append(keys, []byte("1234")...)
-		binary.BigEndian.PutUint32(keys[kIdx:], uint32(keyLen))
-		kIdx += 4
-		keys = append(keys, v...)
-		kIdx += len(v)
-	}
-	for _, v := range labelValues {
-		valLen := len(v)
-		values = append(values, []byte("1234")...)
-		binary.BigEndian.PutUint32(values[vIdx:], uint32(valLen))
-		vIdx += 4
-		values = append(values, v...)
-		vIdx += len(v)
+func pgLabel(id int64) pgtype.Int8 {
```

Comment:
where is this used?

Reply:
it's a leftover from an old rewrite of these tests. Removed.